cnr_scene_manager_test_move:
    n_objs: 5
